recipe/meta.yaml

```diff
@@ -26,8 +20,7 @@ requirements:
     - make        # [unix]
     - pkg-config  # [unix]
     - python
-  host:
-    - msinttypes  # [win and vc==9]
+  host: []
```

Comment:
Need an empty list here due to issue ( https://github.com/conda-forge/conda-smithy/issues/1153 ).